modules/nodes-nodes-cluster-timeout-graceful-shutdown.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: CONCEPT
+[id="nodes-nodes-cluster-timeout-graceful-shutdown_{context}"]
+= About graceful node shutdown
+
+During a graceful node shutdown, the kubelet sends a termination signal to pods running on the node and postpones the node shutdown until all the pods are evicted and drained. If a node unexpectedly shuts down, the graceful node shutdown feature minimizes interruption to workloads running on these pods. 
```

Comment:
Generally, we say drain node and evict pod. So modify "until all the pods are evicted and drained" to "until all the pods are evicted"